blog/2023-3-5-magazine-002.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Inflight Magazine no. 2
+description: Updates from the Wing team
+authors: 
+  - eladb
+  - name: Revital Barletz
+    title: Head of community
+    url: https://github.com/revitalbarletz
+    image_url: https://avatars.githubusercontent.com/revitalbarletz
+
+tags: [cloud-oriented programming, winglang, newsletter, updates, inflights, community]
+hide_table_of_contents: true
+---
+
+Hi there!
+
+We're excited to share with you the second issue of the **Wing Inflight Magazine** 
+([here's a link to our first issue if you missed it](https://docs.winglang.io/blog/2022/12/28/magazine-001)).
+
+The **Inflight Magazine** is where you can stay up to date with [Wing Programming Language](https://winglang.io) developments and the awesome community that is forming around it. 
+
+You received this email because we have you on our list as someone who might be interested to stay informed about the Wing Programming Language.
+
+As always, we would love to hear what you think. Feel free to reply directly to this email, mention [@winglangio](https://twitter.com/winglangio) on Twitter
+or ping us on [slack](https://t.winglang.io/slack).
+
+<!--truncate-->
+
+## TL;DR
+
+Here is a summary of the topics in today's issue:
+
+ - 🔍 [Check out the latest Wingly Update episode](#check-out-the-latest-wingly-update-episode) 
+ - 🔍 [Wing User Stories are now published to our doc site](#wing-user-stories-are-now-published-to-our-doc-site)
+ - 🔍 [Wing Console​ gets a new map view](#wing-console-gets-a-new-map-view) - New look and feel for the wing console application view based on ELK algorithm
+ - 🔍 Deep dive: user resources  - Create reusable building blocks organized into logical units that abstract both functional logic and infrastructure
+ - 🚀 New: Introducing duration and datetime types - our primary engineering focus is to finish implementing the compiler so we can actually start using Wing to build some real stuff
+ - 🚀 New: json type support  - First class support for JSON in Wing
+ - 🚀 New: copy and copy_mut  - Convert mutable and immutable types of wing primitive collections
+ - 🗣️ Updates to our contribution guide - Easier navigation through the guide and accessibility to  day-to-day workflows
+
```

Comment:
Can we share that we are hiring, please?
https://www.linkedin.com/jobs/view/3482716147/

Reply:
Yes, also doc writer!